fix: correctly parse multistage dockerfiles written in lowercase

RotemS opened the pull request:
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Bumping snyk-docker-plugin (contains a fix for parsing dockerfiles written in lowercase)

dkontorovskyy commented on package.json:
```diff
@@ -78,7 +78,7 @@
     "semver": "^6.0.0",
     "snyk-config": "^2.2.1",
-    "snyk-docker-plugin": "1.33.1",
+    "snyk-docker-plugin": "1.37.3",
```
There is already `1.38.0`, maybe upgrade to that version?

RotemS replied:
You're right, I missed it 🙈 Changed

snyksec commented:
<!--
  0 failure: 
  0 warning: 
  1 messages
  
  DangerID: danger-id-Danger;
-->



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  This PR will not trigger a new version. It doesn't include any commit message with `feat` or `fix`.
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ab396d301e871765988b4b5cd1e75b6b9b7cf772
</p>

lili2311 commented:
@RotemS please change the commit to `feat` this is a change that release new features, `chore` will not trigger a release. Please could you also update the commit message to explain what will be included in this new release? The commit is part of release notes, currently it doesn't explain

RotemS commented:
> @RotemS please change the commit to `feat` this is a change that release new features, `chore` will not trigger a release. Please could you also update the commit message to explain what will be included in this new release? The commit is part of release notes, currently it doesn't explain

@lili2311 hope it's ok, changed to `fix` as I thought it was more suitable. Also updated the commit message.

snyksec commented:
:tada: This PR is included in version 1.283.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/snyk/v/1.283.0)
- [GitHub release](https://github.com/snyk/snyk/releases/tag/v1.283.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
